types/google.analytics/index.d.ts

```diff
@@ -39,8 +40,17 @@ interface GoogleAnalytics {
 declare namespace UniversalAnalytics {
     // https://developers.google.com/analytics/devguides/collection/analyticsjs/method-reference
 
-    enum HitType {
-        'pageview', 'screenview', 'event', 'transaction', 'item', 'social', 'exception', 'timing'
+    // https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#hitType
+    type HitType = 'pageview' | 'screenview' | 'event' | 'transaction' | 'item' | 'social' | 'exception' | 'timing';
+    enum HitEnum {
```

Comment:
Is `HitEnum` needed? It seems to be unused.

Reply:
Yap. It might be redundant. But the origin HitType in enum seems reasonable, that's why I tried to keep the enum there. Also, even we can use the enum to ensure the typo-free, the enum type doesn't match the `HitType` definition...

Thanks for mention this part, I'm confused about should we keep the enum exists?

Reply:
I think it's fine to remove the `HitEnum`. `HitType` on its own should prevent issues, since typescript will throw an error if a string that's not listed is used for something with type `HitType`.